filer/views.py

```diff
@@ -21,7 +22,11 @@ def selectfolder_param(request, separator="&"):
         return ""
 
 def current_site_param(request, separator="&"):
-    if 'current_site' in request.REQUEST:
-        return '%scurrent_site=%s' % (separator,
-                                      request.REQUEST['current_site'])
+    current_site = get_param_from_request(request, 'current_site')
+    if current_site:
+        return '%scurrent_site=%s' % (separator, current_site)
     return ""
+
+
+def get_param_from_request(request, param, default=None):
+    return request.POST.get(param) or request.GET.get(param) or default
```

Comment:
by the fact we try to get the param, no matter which method is used, aren't we kind of hiding possible issues?